compute/disks/create_hyperdisk_from_pool.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "cloud.google.com/go/compute/apiv1/computepb"
+	"google.golang.org/protobuf/proto"
+)
+
+// [START compute_hyperdisk_create_from_pool]
+
+// createDiskInStoragePool creates a new Hyperdisk in the specified storage pool.
+func createDiskInStoragePool(w io.Writer, projectId, zone, diskName, storagePoolName, diskType string, diskSizeGb, provisionedIops, provisionedThroughput int64) error {
```

Comment:
please simplify this function signature by removing the size, IOps and Throughput parameters.